src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -915,7 +932,11 @@ BOOST_AUTO_TEST_CASE(GradientGRULayerTest)
       target = arma::ones(1, 1, 5);
       const size_t rho = 5;
 
-      model = new RNN<NegativeLogLikelihood<> >(input, target, rho);
+      model = new RNN<NegativeLogLikelihood<> >(rho);
+      arma::cube& temp = model->Predictors();
```

Comment:
We can use: ``model->Predictors() = input;`` and ``model->Responses() = target;``.

Reply:
Done